src/lib.rs

```diff
@@ -1,16 +1,15 @@
 /*
-Copyright (c) 2023 Uber Technologies, Inc.
+  Copyright (c) 2023 Uber Technologies, Inc.
```

Comment:
for some reason our license linter wanted me to add these white spaces. :| anyhoo